release/models/macsec/openconfig-macsec.yang

```diff
@@ -18,11 +18,17 @@ module openconfig-macsec {
     "This module defines configuration and state data for
      MACsec IEEE Std 802.1AE-2018.";
 
-  oc-ext:openconfig-version "1.0.0";
+  oc-ext:openconfig-version "1.0.1";
```

Comment:
```suggestion
  oc-ext:openconfig-version "1.1.0";
```

Need to update minor version due to addition of leaf.

Reply:
Thanks for the review. I have updated the minor version in the latest update.